app/src/main/kotlin/co/netguru/baby/monitor/client/application/di/ApplicationModule.kt

```diff
@@ -79,4 +81,11 @@ object ApplicationModule {
             database.execSQL("ALTER TABLE CLIENT_DATA ADD COLUMN voiceAnalysisOption TEXT NOT NULL DEFAULT ${VoiceAnalysisOption.MACHINE_LEARNING.name}")
         }
     }
+
+    private val MIGRATION_4_5 = object : Migration(4, 5) {
+        override fun migrate(database: SupportSQLiteDatabase) {
+            database.execSQL("ALTER TABLE CHILD_DATA ADD COLUMN noiseSensitivity INTEGER NOT NULL DEFAULT ${NoiseDetector.DEFAULT_NOISE_SENSITIVITY}")
+            database.execSQL("ALTER TABLE CLIENT_DATA ADD COLUMN noiseSensitivity INTEGER NOT NULL DEFAULT ${NoiseDetector.DEFAULT_NOISE_SENSITIVITY}")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/c7380d74fc32b4ddceef02c8d9a561008c986f4f/app/src/main/kotlin/co/netguru/baby/monitor/client/application/di/ApplicationModule.kt#L88"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>